wandb/sync/sync.py

```diff
@@ -140,26 +149,58 @@ def _send_tensorboard(self, tb_root, tb_logdirs, send_manager):
         )
         print("Syncing: %s ..." % url)
         sys.stdout.flush()
+        # this sucks, but if we want to add in the Handler specifically for tb case
+        # we need to remake the SendManager and Backend Sender
+        record_q = queue.Queue()
+        sender_record_q = queue.Queue()
+        new_interface = interface.BackendSender(record_q)
+        send_manager = sender.SendManager(
+            send_manager._settings, sender_record_q, queue.Queue(), new_interface
+        )
```

Comment:
This does suck.  Why do we "want to add in the Handler"?  Let's at least add a comment explaining why we are doing this.

Reply:
The reason we want to add the handler is that it populates the summaries, handles different file types, and keeps the step issues without having to repeat all of that logic here. I can add a comment